### PR TITLE
[rest api] fix purchases documentation

### PR DESCRIPTION
**Pull request by emizzle**
Documentation in the REST OpenAPI spec were showing incorrect information about Purchasing endpoints.

Update the open api spec to reflect information about purchases, not requests.

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -91,6 +91,18 @@ components:
...
+        properties:
+          state:
+            type: string
+            description: Description of the Request's state
+          error:
+            type: string
+            description: If Request failed, then here is presented the error message
+          request:
+            $ref: "#/components/schemas/StorageRequest"
+
```
Comment (AuHau) [inline openapi.yaml:105]: I think this is duplicate as you have renamed `StorageRequestState` into `Purchase` [here](https://github.com/status-im/nim-codex/pull/380/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R180)?

**Review by AuHau — COMMENTED** [review]
Thanks a lot! 🙏 Generally LGTM, but I think there is one component duplicated...

**Review by markspanbroek — DISMISSED** [review]

File: openapi.yaml (hunk 1 above)
Comment (emizzle) [reply openapi.yaml]: Good catch! Duplicate removed.

**Review by AuHau — APPROVED** [review]
LGTM, thanks!

**Pull request merged**